CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## v0.2.0
```

Comment:
is this considered a breaking change do we think? If so does that mean we move to 1.0.0? I feel like @iwysiu probably knows semver better than me haha.

Reply:
v0 s are pre-release development versions, so we don't need to bump it. A 1.0.0 would be when we cut the first actual release!